src/src/rx-serial/SerialHoTT_TLM.h

```diff
@@ -252,18 +252,27 @@ typedef struct
     bool present;
 } hottDevice_t;
 
+enum {
+    HOTT_RECEIVING,
+    HOTT_CMD1SENT,
+    HOTT_CMD2SENT
+};
 
 class SerialHoTT_TLM : public SerialIO
 {
 public:
     explicit SerialHoTT_TLM(Stream &out, Stream &in)
         : SerialIO(&out, &in)
     {
+        // use UART0 default rx pin for half duplex if not defined otherwise
+        rxPin = GPIO_PIN_RCSIGNAL_RX == UNDEF_PIN ? 3 : GPIO_PIN_RCSIGNAL_RX;
```

Comment:
3 is a magic number and will only work for ESP32 and 8285.
I think I created a define for this... maybe it's in the C3 PR... yeah, maybe put the part below into this PR and use the `U0RXD_GPIO_NUM` define.
![Screenshot 2024-04-17 at 7 50 46 AM](https://github.com/ExpressLRS/ExpressLRS/assets/512740/f3fef68a-433d-424d-983f-f9be3f57341b)

Reply:
Awesome, I was searching for a define but it's difficult to search for "3" or guess how it could be named.